test/test-imports/java/test.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+set -e
+scriptdir=$(cd $(dirname $0) && pwd)
+
+cd $(mktemp -d)
+mkdir test && cd test
+
+cp ${scriptdir}/cdk8s.yaml  .
+cp ${scriptdir}/mattermost_crd.yaml .
+cp ${scriptdir}/example_multiple_crd.yaml .
+
+cdk8s import mattermost:=mattermost_crd.yaml --language java
+
+for f in `find imports/*`
+do
+    diff -I '^@javax.annotation.Generated' $f ${scriptdir}/expected-named-from-cli/${f#*/}
+done
```

Comment:
@eladb I could not get `grep -v` working, however, this way of diffing worked for me. Let me know if it's alright!